shared/gast_to_code/gast_to_code_router.py

```diff
@@ -13,6 +13,9 @@ def gast_to_code(gast, out_lang, lvl=0):
     converter = ConverterRegistry.get_converter(out_lang)
     
     if type(gast) == list:
+        # bash has no comma between function arguments
+        if out_lang == "bash":
```

Comment:
Is there an enum of languages, this looks like the magic string problem there used to be in gast.

Reply:
+1